### PR TITLE
switch command syntax from juju 2.x to juju 3.x styles

### PR DESCRIPTION
**Pull request by addyess**
Changing command syntax from juju 2.x to juju 3.x style based on the [juju release-notes](https://discourse.charmhub.io/t/roadmap-releases/5064#heading--juju-3-0-0---22-oct-2022)

I left the historical notes in place, but only the current notes will take adjustment

Major changes are around these client changes
| Original | New | 
| --- | --- | 
| juju run    |   juju exec| 
| juju run-action --wait |  juju run | 
| juju relate | juju integrate | 
| juju add-relation | juju integrate |

### CODE REVIEW

**Comment by evilnick** [comment]
Nice. I think this also now needs to explicitly call out that we require 3.x 
Probably on the install-manual page and the quickstart page. I will take a look

File: pages/k8s/troubleshooting.md [hunk 1]
```diff
@@ -267,7 +267,7 @@ This is caused by the API load balancer not forwarding ports in the context of t
     If you want to access this data programmatically you can use the JSON output:
 
     ```bash
-    juju show-status kubernetes-control-plane --format json | jq --raw-output '.applications."kubernetes-control-plane".units | keys[]'
+    juju status --format json | jq -r '.applications."kubernetes-control-plane".units | to_entries[] | [.value."public-address"] | @tsv'
```
Comment (kwmonroe) [inline pages/k8s/troubleshooting.md:270]: wow, nice spot

File: pages/k8s/upgrade-notes.md [hunk 2]
```diff
@@ -206,7 +206,7 @@ juju add-relation docker kubernetes-worker
 Now, pause the existing workers, which will move pods to the temporary workers.
 
 ```bash
-juju run-action [unit] pause --wait
+juju run-action [unit] pause
```
Comment (kwmonroe) [inline pages/k8s/upgrade-notes.md:209]:
```suggestion
juju run [unit] pause
```
Gotta catch 'em all

File: pages/k8s/validation.md [hunk 3]
```diff
@@ -65,11 +65,17 @@ It may take some moments for these relations to establish. Once the connections
 The tests are configured as a **Juju** _action_. To run the default tests:
 
 ```bash
-juju run-action kubernetes-e2e/0 test
+juju run kubernetes-e2e/0 test --background
 ```
 
-The command will return with a uuid for that specific test run. See the section
-on _Test output_ below for details.
+The command will return with a number for that specific action operation.
```
Comment (kwmonroe) [inline pages/k8s/validation.md:71]: :heart:

**Review by kwmonroe — COMMENTED** [review]
Looking good.  Eyes crossed a few times, but i think you just missed one `s/run-action/run`.

File: pages/k8s/upgrade-notes.md (hunk 2 above)
Comment (kwmonroe) [reply pages/k8s/upgrade-notes.md:209]: ah, ok then

**Review by kwmonroe — APPROVED** [review]

**Comment by evilnick** [comment]
i made a change to the quickstart page (which installed 2.9). Sad that 3.1 isn't the default Juju...
I will make a further issue for adding additional notifications, but at least this stops the docs being wrong. thanks!

**Pull request merged**